### PR TITLE
feat(EMS-640): Your Business - Nature of your business - years exporting field

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:
* Added years exporting to nature of your business page and validation
* Added tests
* Fixed TS error with maximum for goods or services field

### CODE REVIEW

File: e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/nature-of-business-page.spec.js [hunk 1]
```diff
@@ -25,13 +30,15 @@ context('Insurance - Your business - Nature of your business page - As an Export
 
     cy.submitInsuranceEligibilityAndStartApplication();
 
+    task.link().click();
+
+    cy.completeCompanyDetails();
```
Comment (ttbarnes) [inline e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/nature-of-business-page.spec.js:35]: Do you think we should have `completeAndSubmit` prefix? Done this in other pages but happy to not do this

File: e2e-tests/cypress/support/insurance/complete-company-details.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { companyDetails } from '../../e2e/pages/your-business';
+import { submitButton, yesRadioInput } from '../../e2e/pages/shared';
+import { COMPANIES_HOUSE_NUMBER } from '../../../constants';
+
+export default () => {
+  companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+  yesRadioInput().first().click();
+  yesRadioInput().eq(1).click();
```
Comment (ttbarnes) [inline e2e-tests/cypress/support/insurance/complete-company-details.js:8]: Are we going to have more descriptive radios? Think we mentioned in a previous PR, concious of more instances of this being introduced. Could have e.g `xFieldYesRadioInput`  Not a blocker

File: src/ui/server/helpers/number/index.ts [hunk 3]
```diff
@@ -23,4 +23,6 @@ const numberHasDecimal = (value: number) => String(value).includes('.');
  */
 const getPercentageOfNumber = (percent: number, total: number) => ((percent / 100) * total).toFixed(2);
 
-export { isNumber, numberHasDecimal, getPercentageOfNumber };
+const isNumberBelowMinimum = (value: number | string, minimum: number | string) => Number(value) < Number(minimum);
```
Comment (ttbarnes) [inline src/ui/server/helpers/number/index.ts:26]: documentation 

Also can we enforce `number` type?

File: e2e-tests/cypress/support/insurance/complete-company-details.js (hunk 2 above)
Comment (Zainzzkk) [reply e2e-tests/cypress/support/insurance/complete-company-details.js:8]: how would we do this as we have a lot of files using `yesRadioInput`

Comment (ttbarnes) [reply e2e-tests/cypress/support/insurance/complete-company-details.js:8]: As part of `companyDetailsPage`

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**